packages/compiler/src/plugins.ts

```diff
@@ -77,6 +78,27 @@ export const processMyst = () => {
           };
           (parent as Parent).children[index] = newNode;
           return index;
+        } else if (mystType === "panels") {
+          let cards = parsePanels(value);
+          console.log(cards);
+          // parse each card
+          const newNode = {
+            type: "html",
+            value: mustache.render(
```

Comment:
I think we want to put this template in a seperate file, maybe something like `PanelDefinition.svelte`?

Reply:
I'm not sure exactly where you want to separate this, i.e. based on value, etc. (so I haven't broken it out yet). Do you want to have mustache templates in their own file that gets loaded for each type of MyST object?

Reply:
Ok, will leave as is for now.